flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaTableITCase.java

```diff
@@ -1514,4 +1510,22 @@ private static boolean isCausedByJobFinished(Throwable e) {
             return false;
         }
     }
+
+    private void cleanupTopic(String topic) {
+        ignoreExceptions(() -> deleteTestTopic(topic));
+    }
+
+    private static void ignoreExceptions(RunnableWithException e) {
+        try {
+            e.run();
+        } catch (Exception ex) {
+            // ignore
+        }
+    }
+
+    private static void cancelJob(TableResult tableResult) {
+        if (tableResult != null) {
+            ignoreExceptions(() -> tableResult.getJobClient().ifPresent(JobClient::cancel));
+        }
```

Comment:
This only looks like a temporary solution. Can we at least file/link a ticket to fix it properly?

Reply:
Why do you think it's temporary? Cleanup code can be invoked at any point in time during test lifecycle. So job may have already failed or not. In the former case, JobClient::cancel throws. In the latter case, it's needed.